packages/vorbis/src/common.rs

```diff
@@ -0,0 +1,16 @@
+use std::mem::MaybeUninit;
+
+#[macro_use]
+mod error;
+pub use error::VorbisError;
+
+mod ogg_data;
+pub use ogg_data::{OggPacket, OggPage, OggStream};
```

Comment:
Good eye on `OggPage`! didn't notice that!

I agree with you on `ogg_data` and `vorbis_data` aswell, but for continuity's sake I think it'd fit if `encoder_data` and `decoder_data` in their respective directories were renamed aswell (because they also contain abstractions), but if i were to change them to say `decoder`/`encoder` following the `ogg` and `vorbis` pattern then clippy would throw a module inception warning, what do you reckon I should do for those? or should they be kept as is because they're higher level abstractions?

Reply:
I thought about that while reviewing your changes too! My opinion was that, even though that `encoder_data` and `decoder_data` do not contain data structures, they are more like "mixed bags" for stuff related to the encoder and decoder, so the current name is somewhat acceptable. Now that I think more about it, though, what's your opinion on naming them something like `{en,de}coder_aux` or `{en,de}coder_util`?

Reply:
I think `{en,de}coder_util` would fit pretty nicely, `util` by itself is a pretty common suffix and the stuff in them is used as utilities for the actual implementation; I'll rename them to that!